src/components/Home.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import logo from '../images/europe.svg';
+import CountryList from './CountryList';
+import Filter from './input/Filter';
+import Search from './input/Search';
+
+const Home = () => (
+  <div className="flex flex-col p-4">
+    <div className="flex flex-col items-center">
+      <p className="w-full text-center text-white font-bold text-2xl">Europe</p>
+      <div className="grid place-items-center w-full">
+        <img src={logo} className="w-1/2" alt="Logo" />
+      </div>
+      <p className="w-full text-center text-white font-bold text-xl mb-4">All Countries</p>
+    </div>
+    <div className="mb-8 flex flex-col md:flex-rowjustify-between">
+      <Search />
+      <Filter />
```

Comment:
Make these 2 the same width right now it looks bad like this!

![Screenshot from 2023-10-13 11-50-48](https://github.com/memicelis/countries/assets/118063058/f647e47b-4779-4587-805a-456f6ab5780a)

On the desktop version make sure you do better styling of it as well, right now they don't have any margins or space between them, you can position them in line and do space between for desktop version. 

![Screenshot from 2023-10-13 11-51-38](https://github.com/memicelis/countries/assets/118063058/a2fadd3b-2791-4468-a465-ae4c0254f572)